app/views/events/finance.html.erb

```diff
@@ -8,18 +8,26 @@
       <% if not ed.timecard_entries.empty? %>
         <tr>
           <th><%= ed.description%></th>
+          <th>Event Role</th>
+          <th>Subevent Role</th>
```

Comment:
Small nitpick, call this "Event roles" and  "Event date roles" to be consistent with rest of Tracker. I think plural because there can be multiple.

Reply:
Agreed that event date is more consistent, however I think event date is confusing for people without internal knowledge. Perhaps it can be renamed/refactored in the future.

Reply:
Ah, makes sense. I still find event date a little unintuitive compared to subevent, but that's probably because I'm not opposed to thinking of components of a single event like loadin, show as an event in itself.